lib/active_storage_validations/size_validator.rb

```diff
@@ -31,7 +31,8 @@ def validate_each(record, attribute, _value)
         errors_options[:file_size] = number_to_human_size(file.blob.byte_size)
         errors_options[:min_size] = number_to_human_size(min_size(flat_options))
         errors_options[:max_size] = number_to_human_size(max_size(flat_options))
-        error_type = "file_size_not_#{flat_options.keys.first}".to_sym
+        keys = AVAILABLE_CHECKS & options.keys
```

Comment:
@jozzi05 could you use the `flat_options` variable instead of `options` here? =>  `AVAILABLE_CHECKS & flat_options.keys`

Reply:
https://github.com/igorkasyanchuk/active_storage_validations/pull/209/commits/5ad49da491b0e1a02373c66929a2cdbd82c56e4a